canvas_modules/common-canvas/src/common-properties/components/properties-buttons/properties-buttons.scss

```diff
@@ -17,7 +17,6 @@
 .properties-modal-buttons {
 	justify-content: flex-end;
```

Comment:
I think we might able to remove left, justify-content, and display.

Reply:
Removed `left: 0` style. For justify-content and display, button height and position changes after removing these styles.
![Button styles adjustment](https://user-images.githubusercontent.com/25124000/95510044-2d87f400-096a-11eb-9260-d1517f9db289.gif)